tests/ops/email_templates/test_get_email_template.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from jinja2 import Template
+
+from fidesops.ops.common_exceptions import EmailTemplateUnhandledActionType
+from fidesops.ops.email_templates import get_email_template
+from fidesops.ops.schemas.email.email import EmailActionType
+
+
+def test_get_email_template_returns_template():
+    result = get_email_template(EmailActionType.SUBJECT_IDENTITY_VERIFICATION)
+    assert type(result) == Template
+
+
+def test_get_email_template_exception():
+    fake_template = "templateThatDoesNotExist"
```

Comment:
This is technically a type error instead of a functionality test, but since you can't change an enum and it only has 1 value I'm not sure you can do better here.

Reply:
Your reasoning makes sense. In this scenario you described the type check wouldn't catch this so I think the error is good. I am surprised that mypy isn't complaining that no `Template` is returned, but it seems happy and I can see it will always work as expected so seems good.

Reply:
I think we may not run mypy on the `tests` directory in the codebase. I tried running it as a test and it's pulling up typing issues.